content/applications/productivity/calendar/google.rst

```diff
@@ -197,3 +197,63 @@ The following reset options are available for troubleshooting Google calendar sy
 
 Click :guilabel:`Confirm` after making the selection to modify the user's events and the calendar
 synchronization.
+
+Google OAuth FAQ
+================
+
+At times there can be misconfigurations that take place, and troubleshooting is needed to resolve
+the issue. Below are the most common errors that may occur when configuring the *Google Calendar*
+for use with Odoo.
+
+Production vs. Testing Publishing Status
+----------------------------------------
+
+Choosing :guilabel:`Production` as the :guilabel:`Publishing Status` (instead of
+:guilabel:`Testing`) displays the following warning message:
+
+.. image:: google/published-status.png
+   :align: center
+   :alt: OAuth is Limited to 100 Sensitive Scope Logins.
```

Comment:
This instruction is relying on the image to explain information, lets remove that dependency by writing out (in a :guilabel:) the on-screen text that is displayed. The image should always be supplemental to text.
```suggestion
Choosing :guilabel:`Production` as the :guilabel:`Publishing Status` (instead of
:guilabel:`Testing`) displays the following warning message:

.. image:: google/published-status.png
   :align: center
   :alt: OAuth is Limited to 100 Sensitive Scope Logins.
```